proposals/boost/boost-maven/boost-maven-plugin/src/it/test-mpRestClient/src/main/java/io/openliberty/guides/inventory/InventoryManager.java

```diff
@@ -0,0 +1,254 @@
+// tag::copyright[]
+/*******************************************************************************
+ * Copyright (c) 2017, 2018 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - Initial implementation
+ *******************************************************************************/
+// end::copyright[]
+// tag::manager[]
+package io.openliberty.guides.inventory;
+
+import java.net.ConnectException;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.net.UnknownHostException;
+import java.util.ArrayList;
+import java.util.Calendar;
+import java.util.Collections;
+import java.util.Date;
+import java.util.List;
+import java.util.Properties;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.enterprise.context.Dependent;
+import javax.inject.Inject;
+import javax.ws.rs.ProcessingException;
+
+import org.apache.commons.lang3.exception.ExceptionUtils;
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+import org.eclipse.microprofile.rest.client.RestClientBuilder;
+import org.eclipse.microprofile.rest.client.inject.RestClient;
+
+import io.openliberty.guides.inventory.client.SystemClient;
+import io.openliberty.guides.inventory.client.UnknownUrlException;
+import io.openliberty.guides.inventory.client.UnknownUrlExceptionMapper;
+import io.openliberty.guides.inventory.model.InventoryList;
+import io.openliberty.guides.inventory.model.InvokeTracker;
+import io.openliberty.guides.inventory.model.SystemData;
+
+import javax.transaction.HeuristicMixedException;
+import javax.transaction.HeuristicRollbackException;
+import javax.transaction.NotSupportedException;
+import javax.transaction.RollbackException;
+import javax.transaction.SystemException;
+import javax.transaction.UserTransaction;
+import javax.naming.Context;
+import javax.naming.InitialContext;
+import javax.naming.NamingException;
+import javax.persistence.EntityManager;
+import javax.persistence.PersistenceContext;
+
+import java.util.logging.Logger;
+
+@ApplicationScoped
+public class InventoryManager {
+
+    private List<SystemData> systems = Collections.synchronizedList(new ArrayList<>());
+
+    private static String newline = System.getProperty("line.separator");
+
+    Logger LOG = Logger.getLogger(InventoryManager.class.getName());
+
+    @PersistenceContext(unitName = "guideapp-persister")
+    private EntityManager em;
+
```

Comment:
Is the persistence validated by the test logic?   (I didn't try to read the code, just figured I'd ask.)

Reply:
removed all jpa/jdbc logic